src/main/java/com/coveo/pushapiclient/Source.java

```diff
@@ -15,18 +15,48 @@ public class Source {
    * @param organizationId The Coveo Organization identifier.
    */
   public Source(String apiKey, String organizationId) {
-    this.platformClient = new PlatformClient(apiKey, organizationId);
+    this(apiKey, organizationId, new BackoffOptionsBuilder().build());
   }
 
   /**
    * @param apiKey An apiKey capable of pushing documents and managing sources in a Coveo
    *     organization.
    * @see <a href="https://docs.coveo.com/en/1718">Manage API Keys</a>.
    * @param organizationId The Coveo Organization identifier.
-   * @param platformUrl
+   * @param options The options for exponential backoff.
```

Comment:
This class will soon be deprecated. It should not have any additional constructors.
It will be replaced by PushSource

Reply:
Good point! I'll revert this change 🙂